### ISSUE TITLE
Fresh install does not work out of the box

### ISSUE THREAD

**Issue by Bachibouzouk** [rl-institut/WAM_APP_stemp_abw#95]
When following the steps for stemp_abw [installation](https://stemp-abw.readthedocs.io/en/dev/installation_and_setup.html#install-data-label), I get the following upon

`python manage.py runserver`

```
django.db.utils.ProgrammingError: relation "stemp_abw_scenario" does not exist
LINE 1: ...stemp_abw_scenario"."repowering_scenario_id" FROM "stemp_abw...
```

**Comment by 4lm** [comment]
Hi @Bachibouzouk this is fixed on branch `feature/fixtures-install` https://github.com/rl-institut/WAM_APP_stemp_abw/tree/feature/fixtures-install but not yet merged into dev #77

**Comment by Bachibouzouk** [comment]
Duplicate of #71
